cmake/extensions.cmake

```diff
@@ -33,3 +33,21 @@ function(add_overlay_config image overlay_file)
   endif()
 endfunction()
 
+# Add a partition manager configuration file to the build.
+# Note that is only one image is included in the build,
+# you must set CONFIG_PM_SINGLE_IMAGE=y for the partition manager
+# configuration to take effect.
+function(add_partition_manager_config config_file)
```

Comment:
I know this haven't been a principle before, so feel free to ignore for now, as we can do it at later cleanup (which we would anyway need to do for other functions).

But should we start to introduce a `ncs_<function_name>` for customer faced CMake extension functions ?

Reply:
I think that is a good idea. Lets start here :)